src/Responses/ArmMetadata.php

```diff
@@ -1,47 +1,45 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Keboola\AzureKeyVaultClient\Responses;
 
 use Keboola\AzureKeyVaultClient\Exception\InvalidResponseException;
 
 class ArmMetadata
 {
-    /** @var string */
-    private $loginEndpoint;
-    /** @var string */
-    private $keyVaultDns;
-    /** @var string */
-    private $name;
+    private string $loginEndpoint;
+    // @phpstan-ignore-next-line
```

Comment:
Co se tu nelibi?

Reply:
ty props nejsou použitý a nechci je mazat, vím, že to bylo nějaký zamotaný a "správně" by se používat měly, ale nechcu do toho zabrušovat a předělávat autorizaci když funguje

Reply:
ok